graphql_auth/signals.py

```diff
@@ -1,11 +1,16 @@
 from django.db.models.signals import post_save
-from django.dispatch import receiver
 from django.conf import settings as django_settings
 
-from .models import UserStatus
+from django.dispatch import Signal, receiver
 
 
 @receiver(post_save, sender=django_settings.AUTH_USER_MODEL)
 def create_user_status(sender, instance, created, **kwargs):
     if created:
+        from .models import UserStatus
```

Comment:
@PedroBern This line was moved here to prevent a circular import (in other words: the import was not removed), since `models.py` needs `signals.py` and vice versa. It's not ideal but the alternative is to put the new signals in a different file, and I think that's worse (giving that file a proper name is difficult, and moreover, I think a user would expect to find all signals in the `signals.py` file). Let me know what you think!

Reply:
My mistake I missed the line! It's perfect then!